File: services/project/models/project.go
package models

// Project struct
type Project struct {
	PID    string `json:"pid"`
	UID    string `json:"uid"`
	Client string `json:"client"`
	Sector string `json:"sector"`
	Name   string `json:"name"`
	Desc   string `json:"desc"`
	UA     string `json:"ua"`
	Base
}

// TableName set database table name
func (Project) TableName() string {
	return "Projects"
}

File: Api/Domain/Projects/Project.cs
using Api.Domain.Projects;
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Runtime.Serialization;
using Version = Api.Domain.Versions.Version;

namespace Api.Domain.Projects
{
    [Table("Project")]
    public class Project
    {

        public Project()
        {
            CreatedAt = DateTime.UtcNow;
        }

        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        [JsonProperty(PropertyName = "projectId")]
        public int PId { get; set; }
        [JsonProperty(PropertyName = "orderId")]
        public int OId {get; set;}
        public string Name { get; set; }
        public string Description { get; set; }
        public string Manager { get; set; }
        public string Client { get; set; }
        public string Sector { get; set; }
        [DatabaseGenerated(DatabaseGeneratedOption.Identity), DataMember]
        public System.DateTime? CreatedAt { get; private set; }
        public virtual ICollection<ProjectHistory> ProjectHistory { get; set; }
        public virtual ICollection<Version> Versions { get; set; }
    }
}
